readme.md

```diff
@@ -84,7 +84,7 @@ rails db:migrate
 rails db:seed
 ```
 
```

Comment:
can show them that you can do all these rails steps in one line. don't think they've seen that yet. `rails db:drop db:create db:migrate db:seed`